source/use-nunjucks/index.html.md.erb

```diff
@@ -0,0 +1,74 @@
+weight: 45
+---
+
+# Use Nunjucks
+
+You can use our Nunjucks code to generate the HTML for pages and components.
+
+You can also change the HTML that's generated by passing options into the Nunjucks code.
+
+If you use Nunjucks, your project will be easier to update when a new version of GOV.UK Frontend is released.
+
+To use Nunjucks:
+
+- you must [install GOV.UK Frontend with Node.js package manager (npm)](/installing-with-npm/)
+- your application and any frameworks that support it must use Node.js
+
+You do not need to follow this guidance to [use Nunjucks macros in the GOV.UK Prototype Kit](https://design-system.service.gov.uk/get-started/prototyping/#using-nunjucks-macros).
+
+## Set up Nunjucks and use the page template
+
+1. Make sure you installed Nunjucks when you [installed GOV.UK Frontend with npm](/installing-with-npm).
+
+2. Import the Nunjucks package at the top of your configuration file, for example:
+
+    ```javascript
+    const nunjucks = require('nunjucks')
+    ```
+
+3. Tell Nunjucks where to find the GOV.UK Frontend template and components by adding the following to your configuration file:
+
+    ```javascript
+    nunjucks.configure([
+      "node_modules/govuk-frontend/"
+    ], {
+      autoescape: true
+    })
+    ```
+
+    `autoescape: true` makes the output render as text by default, to help prevent cross-site scripting (XSS) attacks.
+
+4. Use the GOV.UK Frontend template by adding the following at the top of your view file:
+
+    ```javascript
+    {% extends "govuk/template.njk" %}
+    ```
+
+5. Go to the [default page template example](https://design-system.service.gov.uk/styles/page-template/#default) on the GOV.UK Design System website, then copy the Nunjucks code into your view file.
+
+
+Find out how to [change how the page template works](https://design-system.service.gov.uk/styles/page-template/#changing-template-content).
+
+## Adding a component
+
+You can go to any component page on the Design System website, then copy the Nunjucks macro code from the **Nunjucks** tab of any example.
+
+For example, to add the breadcrumbs component to your page, copy the code from the **Nunjucks** tab in the first example on the [breadcrumbs component page](https://design-system.service.gov.uk/components/breadcrumbs/).
+
+## Changing a component
+
+You can use options to change how a component looks or behaves.
+
+For example, use the `text` option to change the text on a [button](https://design-system.service.gov.uk/components/button/):
+
+```javascript
+{{ govukButton({
+    text: "Example button text"
+}) }}
+```
+
+To see the options for a component, select the **Nunjucks** tab of the component example on any Design System website page, then select **Nunjucks macro options**.
+
+You must sanitise any HTML you pass in to Nunjucks macros you’re using in your live application, to protect your website against cross-site scripting (XSS) attacks. Talk to your team’s security specialist about how to do this.
```

Comment:
I think [@vanitabarrett's comment on #70](https://github.com/alphagov/govuk-frontend-docs/pull/70#discussion_r463640030) applies here as well – the chances that there's a clear 'security specialist' to go and talk to for any given team seem slim to me.